linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_data_disk_error_action/rule.yml

```diff
@@ -48,3 +48,16 @@ ocil: |-
     switch to single-user mode, execute a script,
     or halt when the disk errors:
     <pre>disk_error_action single</pre>
+
+template:
+    name: auditd_lineinfile
+    vars:
+        missing_parameter_pass: 'false'
+        parameter: disk_error_action
+        rule_id: {{{ rule_id }}}
```

Comment:
Don't put the rule_id key here. A rule_id variable is available in the template engine by default.